From 5c8cb7bd0503b874ca528eec0608b35b6021748a Mon Sep 17 00:00:00 2001
From: fourmone <eric.fourmont-ext@st.com>
Date: Tue, 7 Jan 2025 11:02:17 +0100
Subject: [PATCH] Revert "net: stmmac: Fix unbalanced IRQ wake disable warning
 on single irq case"

This reverts commit b0a3c915529efeb7a01d9ee672372fa760357511.
---
 drivers/net/ethernet/stmicro/stmmac/stmmac_main.c | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/stmicro/stmmac/stmmac_main.c
+++ b/drivers/net/ethernet/stmicro/stmmac/stmmac_main.c
@@ -3679,7 +3679,6 @@ static int stmmac_request_irq_single(str
 	/* Request the Wake IRQ in case of another line
 	 * is used for WoL
 	 */
-	priv->wol_irq_disabled = true;
 	if (priv->wol_irq > 0 && priv->wol_irq != dev->irq) {
 		ret = request_irq(priv->wol_irq, stmmac_interrupt,
 				  IRQF_SHARED, dev->name, dev);
